src/stats/probs/cdf.rs

```diff
@@ -289,11 +305,15 @@ impl<T: Ord> CDF<T> {
     }
 
     /// Return total probability of the `CDF`.
+    ///
+    /// Time complexity: O(n), where n is the number of `Entry`s in `CDF`.
     pub fn total_prob(&self) -> LogProb {
         self.inner.last().map_or(LogProb::ln_zero(), |e| e.prob)
     }
 
     /// Return maximum a posteriori probability estimate (MAP).
+    ///
+    /// Time complexity: O(n), where n is the number of `Entry`s in `CDF`.
     pub fn map(&self) -> Option<&T> {
```

Comment:
I still think this should be renamed, such that it doesn't get confused with ... `map`... :D
That'd be a breaking change, though
cc @johanneskoester